projects/commudle-admin/src/app/feature-modules/main-newsletters/components/main-newsletter-list/main-newsletter-list.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { MainNewslettersService } from 'projects/commudle-admin/src/app/feature-modules/main-newsletters/services/main-newsletters.service';
+import { IMainNewsletter } from 'projects/shared-models/main-newsletter.model';
+
+@Component({
+  selector: 'app-newsletters',
+  templateUrl: './main-newsletter-list.component.html',
+  styleUrls: ['./main-newsletter-list.component.scss']
+})
+export class MainNewsletterList implements OnInit {
+
+  newsletters: IMainNewsletter[] = [];
+
+  constructor(
+    private mainNewslettersService: MainNewslettersService
+  ) { }
+
+  ngOnInit(): void {
+    this.getPublishedNewsletters();
+  }
+
+  //ALERT: the string must always be in HTML format
```

Comment:
Can there be other type of strings as input for this function?

Reply:
the input will always be in html format